server/src/surveyRoute.test.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import request from 'supertest';
+import * as surveyDBHandlerUtil from './utils/surveyDBHandlerUtil';
+
+describe('Tes survey route', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('check if /api/surveyResults route is open with cosmosDb configs', async () => {
+    const inputData: any = {
+      sessionId: 'test_session_id',
+      callId: 'test_call_id',
+      acsUserId: 'test_acs_user_id',
+      response: true
+    };
+
+    jest.spyOn(surveyDBHandlerUtil, 'createSurveyDBHandler').mockReturnValueOnce(undefined);
+
+    const app = (await import('./app')).default;
```

Comment:
This is an interesting/clever approach. Does this still work if we move these tests to `app.test.ts`?

As much as possible I'd like to follow the "1 file has 1 test file" approach, and putting these tests in `app.test.ts` makes sense to me since `app` is the unit under test.

Reply:
I am not sure if this will work if I move it to `app.test.ts`. If it works, I will move it and remove the file. Otherwise, I will keep it as it is. I will update the comment.